example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_AccountTest.java

```
@@ -276,6 +303,32 @@ public void onAuthEmailSent(OnAuthEmailSent event) {
         }
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe
+    public void onUsernameChanged(OnUsernameChanged event) {
+        AppLog.i(AppLog.T.API, "Received OnUsernameChanged");
+
+        if (event.isError()) {
+            AppLog.i(AppLog.T.API, "OnUsernameChanged has error: " + event.error.type + " - " + event.error.message);
+
+            switch (event.error.type) {
+                case INVALID_ACTION:
+                    assertEquals(mNextEvent, TestEvents.USERNAME_ERROR_INVALID);
+                    mCountDownLatch.countDown();
+                    break;
+                case INVALID_INPUT:
```

We need to throw an error here if we don't expect it to happen. Otherwise the test will pass if we ever get `INVALID_INPUT` error when we expect the call to actually succeed. (Not in it's current form, in it's current form it'll just hang because we never call `mCountDownLatch.countDown();`, but I assume we'll cover the success path)

---

I added the error statement in 006d6242e40e3b05038da66b4c03e695e45824fb.